Code/Tools/Libs/ToolsFoundation/FileSystem/Implementation/Declarations.cpp

```diff
@@ -0,0 +1,16 @@
+#include <ToolsFoundation/ToolsFoundationDLL.h>
+
+#include <ToolsFoundation/FileSystem/Declarations.h>
+
+// clang-format off
+EZ_BEGIN_STATIC_REFLECTED_TYPE(ezFileStatus, ezNoBase, 3, ezRTTIDefaultAllocator<ezFileStatus>)
```

Comment:
This was moved from the asset curator code which previous owned this class. Technically we could change the version as it is only serialized in the cache which is reset with this commit but I think it would be a bad idea to move the version back in those cases in cases.

Reply:
Not important.